cmd/mattermost/commands/plugin.go

```diff
@@ -172,12 +172,12 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 		return errors.New("Unable to list plugins. Error: " + appErr.Error())
 	}
 
-	CommandPrettyPrintln("Listing active plugins")
+	CommandPrettyPrintln("Listing enabled plugins")
 	for _, plugin := range pluginsResp.Active {
```

Comment:
To be consistent, we should also update the `PluginResponse` [here]( https://github.com/mattermost/mattermost-server/blob/master/model/plugins_response.go#L15-L18) so the code has the proper naming: 
```suggestion
 	for _, plugin := range pluginsResp.Enabled {
```

Reply:
Unfortunately that would be a breaking change. I can leave a comment in the code, if you want.